forge/src/executor/inspector/macros.rs

```diff
@@ -0,0 +1,26 @@
+/// Returns from the function on an error, discarding the error.
```

Comment:
Some helper macros for inspectors:

- `try_or_return`: Most of the time we probably don't want to return custom errors ourselves, so this essentially just defers error handling to REVM's `Machine` in the case something went wrong (see `LogCollector` for an example)
- `as_usize_or_return` is from upstream REVM